src/kaleidoscope/Hardware.h

```diff
@@ -55,6 +57,12 @@ namespace kaleidoscope {
  */
 class Hardware {
  public:
+
+  // To satisfy the interface of those methods that allow
+  // for matrix addressing we define default key and led address classes.
+  // Those typedefs are supposed to overridden by derived hardware classes.
+  typedef MatrixAddr<0, 0> KeyAddr;
+  typedef MatrixAddr<0, 0> LEDAddr;
```

Comment:
I believe the style guide prefers `LedAddr` to `LEDAddr`.

Reply:
I stick with `LEDAddr` analogous to e.g. `LEDControl`. Besides, LED is an acronym, thus IMHO every letter deserves its capital. But that corner case is not defined in the stylegide :wink:.